### PR TITLE
Added option to silence progress bar during installation.

### PR DESCRIPTION
**Pull request by DanieleGBX**
| Q             | A
| ------------- | ---
| Bug fix?      |no
| New feature?  | yes
| BC breaks?    |no
| Deprecations? |no
| Fixed tickets | #131
| License       | MIT

### CODE REVIEW

File: src/Command/CKEditorInstallerCommand.php [hunk 1]
```diff
@@ -70,6 +71,12 @@ protected function configure()
                 InputOption::VALUE_IS_ARRAY | InputOption::VALUE_OPTIONAL,
                 'Path to exclude when extracting CKEditor'
             )
+            ->addOption(
+                'no-progress-bar',
+                'nobar',
+                InputOption::VALUE_NONE,
+                'Should I hide the progress bars while downloading?'
```
Comment (kunicmarko20) [inline src/Command/CKEditorInstallerCommand.php]: I don't think this should be a question, also we don't have progress bar just for downloading. What about `Hide the progress bars?`

**Comment by DanieleGBX** [comment]
Few things regarding the --quiet option:
1. It can't be easily invoked in the composer scripts
2. It's excessive, instead of limiting the feedback output to what's manageable in a non-interactive log, it straight up kills everything. For all scripts.

All in all, I think I was quite delicate with the modifications.

File: src/Command/CKEditorInstallerCommand.php (hunk 1 above)
Comment (DanieleGBX) [reply src/Command/CKEditorInstallerCommand.php]: I was following some symfony style found elsewhere, but yeah.

**Comment by DanieleGBX** [comment]
@kunicmarko20 Do we know what's going to replace it?

**Comment by kunicmarko20** [comment]
https://github.com/FriendsOfSymfony/FOSCKEditorBundle/blob/master/docs/usage/ckeditor.rst#composer-script

This will, see how the config is different.

**Comment by DanieleGBX** [comment]
@ElectricMaxxx any eta on a possible review?

**Pull request merged**